cmake/externals/projects_modules/medInria.cmake

```diff
@@ -61,9 +61,10 @@ EP_SetDirectories(${ep}
 ## Define repository where get the sources
 ## #############################################################################
 
-set(git_url ${GITHUB_PREFIX}Inria-Asclepios/medInria-public.git)
-set(git_tag master)
-
+set(tag master)
+if (NOT DEFINED ${ep}_SOURCE_DIR)
```

Comment:
@juliencastelneau this could be an answer to your problem of the medInria repo which reset when you compile again